vortex-btrblocks/src/string.rs

```diff
@@ -167,6 +173,52 @@ impl Scheme for UncompressedScheme {
     }
 }
 
+impl Scheme for VarBinScheme {
+    type StatsType = StringStats;
+    type CodeType = StringCode;
+
+    fn code(&self) -> StringCode {
+        VARBIN_SCHEME
+    }
+
+    fn expected_compression_ratio(
+        &self,
+        stats: &Self::StatsType,
+        is_sample: bool,
+        allowed_cascading: usize,
+        excludes: &[StringCode],
+    ) -> VortexResult<f64> {
+        estimate_compression_ratio_with_sampling(
+            self,
+            stats,
+            is_sample,
+            allowed_cascading,
+            excludes,
+        )
+    }
+
+    fn compress(
+        &self,
+        stats: &Self::StatsType,
+        _is_sample: bool,
+        _allowed_cascading: usize,
+        _excludes: &[StringCode],
+    ) -> VortexResult<ArrayRef> {
+        use vortex_array::accessor::ArrayAccessor;
+
+        let src = stats.source();
+        let mut builder = VarBinBuilder::<i32>::with_capacity(src.len());
+
+        src.with_iterator(|iter| {
+            for value in iter {
+                builder.append(value);
+            }
+        });
```

Comment:
string compression is already bottleneck on writing (mostly FSST), I'm curious how much this will cost us

Reply:
I should be able to calc without a conversion?